### PR TITLE
Proposal for Falco Infra Admins

### PR DESCRIPTION
**Pull request by nibalizer**
Signed-off-by: Spencer Krum <nibz@spencerkrum.com>

### CODE REVIEW

File: proposals/20200925-admins.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+As part of modern open development, the Falco project is dependent on a considerable amount of development infrastructure. CI systems, code hosting systems, community websites, self-run infrastructure, and cloud services are all examples. Whenever possible, the use of role based access control is used to give the correct people access to the correct services, however in many cases a service account is used with a shared username and password. The group needing access and the number of systems to access are both growing and putting some policies in place will help the community scale effectively.
+
+## Goals
+- Define who is an "infra admin"
+- Define the responsibilities of an "infra admin"
+
+
+## Proposal
+
+- Infra admins are the individuals listed in the OWNERS file of falcosecurity/falco.
```
Comment (leodido) [inline proposals/20200925-admins.md:23]:
```suggestion
- Infra admins are the individuals listed in the OWNERS file of falcosecurity/test-infra.
```

The OWNERS file in Falco repository declares the Falco maintainers.

The OWNERS file in this repo (test-infra) declares the Falco Infra maintainers.

Clearly, the two sets could also be disjoint. Thus, I would not use that OWNERS file to declare Falco Infra maintainers too.

File: proposals/20200925-admins.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- All systems involved in the development of Falco need to be managed by the infra admins.
+- The infra admins team will set up a process and tooling for secrets management. All secrets (passwords, tokens, etc) will be kept in this management system.
```
Comment (leodido) [inline proposals/20200925-admins.md:25]: Maybe worth adding or discussing in the next Falco Infra WG call some solutions, so to add the choice hereafter.

**Review by leodido — COMMENTED** [review]
Left some comments Spencer

File: proposals/20200925-admins.md (hunk 2 above)
Comment (nibalizer) [reply proposals/20200925-admins.md:25]: Right. My thought was picking secrets management might slow this PR down so I just wrote that we should do it in some way.

File: proposals/20200925-admins.md (hunk 1 above)
Comment (nibalizer) [reply proposals/20200925-admins.md:23]: Changed

File: proposals/20200925-admins.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- The infra admin team is responsible for responding to system outages, security incidents, and other responsibilities common to any IT support role.
+- Whenever possible, the infra admins should apply the same principles of open development and open governance to the management of the systems. This is sometimes referred to as "Open Infrastructure."
+- While not required, admins are encouraged to use code and tooling to declaratively define  and update the state of the systems. This technique is sometimes referred to as "GitOps." Ideally, systems will be maintained by the systems themselves, with changes going through public code review and manual changes kept to a minimum.
```
Comment (jonahjon) [inline proposals/20200925-admins.md:28]: Love this tenant ^

**Review by jonahjon — DISMISSED** [review]

**Comment by maxgio92** [comment]
I like the encouragement to declare as much as possible to follow GitOps principles

File: proposals/20200925-admins.md (hunk 2 above)
Comment (leodido) [reply proposals/20200925-admins.md:25]: During the last meeting, we ended up deciding (and applying) for 1Password opensource account.

More in the meeting notes :)

**Comment by nibalizer** [comment]
Is this good? ready to merge?

**Comment by maxgio92** [comment]
/lgmt I think that we can specify later the applied secrets managemenet solution, and go on now with the generic guideline about it
Can I /approve it? <3

**Review by fntlnz — APPROVED** [review]

**Comment by fntlnz** [comment]
Approved ! Thanks a lot for this @nibalizer 

Yes @maxgio92 you had the rights to approve but you need to do a review with the Github UI to allow Poiana to merge.

**Pull request merged**